docs/guide/contributing.rst

```diff
@@ -1,73 +1,206 @@
-=================================================
-Welcome to the Write the Docs documentation guide
-=================================================
-
+=========================================
 Contributing to the Write the Docs guide
-----------------------------------------
+=========================================
+
+If you want to contribute an awesome documentation guide, you've come to the right place. 
+Welcome! Read on to understand what our guides are and how you can help.
 
-If you want to help create an awesome documentation guide, you've come to the right place. Welcome! Please read on to understand what we're trying to do here and how you can help make it a better project.
+Ways to contribute
+-------------------
 
-Ways you can contribute
------------------------
+We encourage contributions from anyone regardless of professional or tool experience. 
+Contributions are treated with respect if they respect the current effort and state of 
+our guide.
 
-The guide is meant to address all aspects of software documentation. Depending on what you're comfortable with, here are the basic ways to contribute:
+The guide at Write the Docs is meant to address all aspects of software 
+documentation. There are several ways to contribute to the content:
 
-* Fork, branch, and submit a pull request
-* Submit an issue. You can use issues to suggest new content, or to file bugs against existing content.
+* Fork, branch, and submit a pull request.
+* `Submit an issue <https://github.com/writethedocs/www/issues>`_ to suggest new content or to file bugs against existing content.
 
 Where to start
 --------------
 
-The guide has its origins in the efforts of developers to understand and explain documentation to each other. Since then, Write the Docs has expanded its focus and its community. We've recently reorganized the guide to provide a clearer top-level structure, but there's much work to be done. Feel free to suggest further changes to the organization. Here's where we need your contributions:
+We especially need contributions in these areas:
+
+* :ref:`Tools and best practices for documenting APIs <api-documentation>`.
+* Documentation best practices.
+* Developer tools or docs-as-code toolchains and workflows.
+* High-level discussions of common tools. 
+
+For help finding where a new set of topics best fit into the rest of 
+the guide, ask in Slack or file a GitHub issue.
+
+Guidelines
+-----------
+
+The guide is not a place to plug your favorite toolchain, even if it's open source. 
+Follow these guidelines for discussing tools: 
+
+* Ensure your content is of interest to a general audience.
+* Avoid advocacy.
+* Present specific use cases, how problems were solved, and what worked or didn't work well. 
+* Consider first creating a GitHub issue or contacting one of the guide editors.
```

Comment:
This "first create an issue" might need unpacking for folks new to this?

Reply:
For now, I've at least made the text a link.